build.gradle.kts

```diff
@@ -2,8 +2,8 @@ import org.gradle.api.tasks.testing.logging.TestExceptionFormat
 
 plugins {
    id("kotest-publishing-conventions")
-   kotlin("jvm") version "1.6.21"
-   id("io.qameta.allure") version "2.10.0"
+   alias(libs.plugins.kotlinJvm).apply(true)
+   alias(libs.plugins.allure).apply(true)
```

Comment:
`.apply(true)`  is done by default, can be omitted

Reply:
done